charts/crljanitor/templates/deployment.yaml

```diff
@@ -59,6 +59,21 @@ spec:
             secretKeyRef:
               name: crl-janitor-stairway-db-creds
               key: db
+        - name: PUBSUB_PROJECT_ID
```

Comment:
wdyt about identifying what the pubsub project/topic/subscription are used for in the janitor? Is it worth saying that this is for the track resource input? Or is there only ever going to be one unqualified pubsub thing so that it's obvious?

Reply:
add TRACK_RESOURCE_ as prefix. PTAL. Thanks